README.md

```diff
@@ -68,16 +68,23 @@ sudo dpkg -i ./omi-1.1.0.ssl_100.x64.deb
 ```
 
 
-### Development Environment
+## Development Environment
 
-##### To clone the repository
+### To clone the repository
 
 ```sh
 git clone https://github.com/Microsoft/omi.git
 ```
 
+### Installing build dependencies
 
-##### Building
+#### Centos 7 x64
+```bash
+sudo yum groupinstall 'Development Tools' -y
+sudo yum install pam-devel.x86_64 openssl-devel.x86_64 -y
+```
+
```

Comment:
I don't think this belongs here in it's current form. Please see project [Build-omi](https://github.com/Microsoft/Build-omi), which gives this information.

Perhaps you just need to refer to that if you'd like for build instructions?

Also, why did you change all of the header levels in MD?